test/test_cpp_api_parity.py

```diff
@@ -218,15 +218,15 @@ def _python_arg_to_cpp_arg(self, python_arg):
         elif type(python_arg) == bool:
             return CppArg(type='bool', value=str(python_arg).lower())
         elif type(python_arg) == str:
-            # if `python_arg` is one of the reduction types, we use the corresponding `Reduction::Reduction` enum.
+            # if `python_arg` is one of the reduction types, we use the corresponding `at::Reduction::Reduction` enum.
             if python_arg in ['none', 'mean', 'sum']:
                 if python_arg == 'none':
-                    cpp_arg = 'Reduction::None'
+                    cpp_arg = 'at::Reduction::None'
                 elif python_arg == 'mean':
-                    cpp_arg = 'Reduction::Mean'
+                    cpp_arg = 'at::Reduction::Mean'
                 elif python_arg == 'sum':
-                    cpp_arg = 'Reduction::Sum'
-                return CppArg(type='Reduction::Reduction', value='{}'.format(cpp_arg))
+                    cpp_arg = 'at::Reduction::Sum'
+                return CppArg(type='at::Reduction::Reduction', value='{}'.format(cpp_arg))
```

Comment:
For C++ frontend we encourage using `torch::` namespace instead of `at::`, and it would be great to change all of the use sites here to `torch::`.